src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -178,20 +178,26 @@ private GATKRead applyREVERT_SOFTCLIPPED_BASES(final GATKRead read) {
     /**
      * Given a cigar string, get the number of bases hard or soft clipped at the start
```

Comment:
Is this method description accurate? If not, write new docs explaining what the return value means.